src/libOpenImageIO/icc.cpp

```diff
@@ -309,6 +309,13 @@ decode_icc_profile(cspan<uint8_t> iccdata, ImageSpec& spec, std::string& error)
             return false;
         }
 
+        if (!check_span(iccdata, iccdata.data() + tag.offset,
```

Comment:
Should we remove the conditional immediately above this line since I think it's duplicated by the stronger `check_span` now?

Reply:
Oh, hmmm, I didn't notice that the test was redundant. Sure.

Reply:
updated